Corona-Warn-App/src/main/res/layout/fragment_submission_positive_efgs_consent.xml

```diff
@@ -0,0 +1,147 @@
+                    app:layout_constraintStart_toStartOf="@id/guideline_start"
+                    app:layout_constraintTop_toBottomOf="@+id/submission_positive_efgs_consent_illustration" />
+
+                <include
+                    android:id="@+id/settings_efgs_consent_row"
+                    layout="@layout/include_settings_switch_row"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_normal"
+                    app:enabled="@{true}"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="1.0"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/submission_positive_efgs_consent_headline"
+                    app:showDivider="@{true}"
+                    app:status="@{FormatterSettingsHelper.formatEfgsConsentSwitchStatus(efgsViewModel.isEfgsConsentGranted())}"
+                    app:statusText="@{FormatterSettingsHelper.formatEfgsConsentSwitchStatusText(efgsViewModel.isEfgsConsentGranted())}"
+                    app:subtitle="@{@string/settings_efgs_consent_title}" />
+
+                <include
+                    android:id="@+id/submission_positive_efgs_consent_privacy"
+                    layout="@layout/include_privacy_card"
+                    android:layout_width="@dimen/match_constraint"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/spacing_normal"
+                    app:layout_constraintEnd_toStartOf="@+id/guideline_card_end"
+                    app:layout_constraintStart_toStartOf="@+id/guideline_card_start"
+                    app:layout_constraintTop_toBottomOf="@+id/settings_efgs_consent_row"
+                    app:privacyText="@{@string/submission_positive_efgs_consent_warning_privacy_body}" />
+
+                <include layout="@layout/merge_guidelines_side" />
+
+                <include layout="@layout/merge_guidelines_card" />
+
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </ScrollView>
+
+        <Button
+            android:id="@+id/submission_positive_efgs_consent_button_next"
+            style="@style/buttonPrimary"
+            android:layout_width="@dimen/match_constraint"
+            android:layout_height="wrap_content"
+            android:text="@string/submission_positive_efgs_consent_warning_button"
+            android:textAllCaps="true"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="@id/guideline_end"
+            app:layout_constraintStart_toStartOf="@id/guideline_start"
+            app:layout_constraintTop_toBottomOf="@id/guideline_action" />
+
+        <ProgressBar
+            android:id="@+id/submission_positive_other_warning_spinner"
```

Comment:
this should be submission_positive_efgs_consent_spinner

(maybe efgs will be replaced)